soccer/gameplay/role_assignment.py

```diff
@@ -34,6 +35,8 @@ def __str__(self):
         props.append("chip_pref=" + str(self.chipper_preference_weight))
         if self.required_shell_id != None:
             props.append("required_id=" + str(self.required_shell_id))
+        if self.prohibited_shell_id != None:
+            props.append("prohibited_id=" + std(self.prohibited_shell_id))
```

Comment:
Where is the `std` coming from here, pylint is complaining about it. If it's legit I can fix it.

Reply:
Yeah, it's a typo. I'm trying to remember why we even added the prohibited id